munkipkg

```diff
@@ -639,6 +639,18 @@ def build(project_dir, options):
             if retcode:
                 raise BuildError("Could not remove existing %s" % outputname)
 
+        # remove .DS_Store file from the scripts folder
+        if os.path.exists(os.path.join(build_info['scripts'], ".DS_Store")):
+            display("Removing .DS_Store file from the scripts folder")
+            os.remove(os.path.join(build_info['scripts'], ".DS_Store"))
+
+        # make scripts executable
+        for pkgscript in ("preinstall", "postinstall"):
+            scriptpath = os.path.join(build_info['scripts'], pkgscript)
+            if os.path.exists(scriptpath) and oct(os.stat(scriptpath).st_mode)[-3:] != '755':
```

Comment:
I thought doing a bitwise-and against 0o555 or 0o500 was clearer for most people, but feel free to use the stat constants if you'd like. The next effect will be the same. (It's actually not important to set execute for group and other here, as long as the owner is root, and it is important to make sure the owner can read it!)